crates/ruff_linter/src/rules/pyflakes/rules/invalid_literal_comparisons.rs

```diff
@@ -110,7 +110,7 @@ pub(crate) fn invalid_literal_comparison(
                 } {
                     diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
                         content,
-                        located_op.range + expr.start(),
+                        located_op.range,
                     )));
```

Comment:
The reason this offset exists was because the lexer function used was `lex` instead of `lex_starts_at`.

https://github.com/astral-sh/ruff/blob/b0a751012e2990f86ff58b17157bf86c73e9a64c/crates/ruff_linter/src/rules/pyflakes/rules/invalid_literal_comparisons.rs#L152-L157